lib/emis/responses/get_military_service_episodes_response.rb

```diff
@@ -7,12 +7,13 @@ module EMIS
   module Responses
     class GetMilitaryServiceEpisodesResponse < EMIS::Responses::Response
       def item_tag_name
-        'militaryServiceEpisodeData'
+        'militaryServiceEpisode'
```

Comment:
@kreek the `personnelCategoryTypeCode` field (that holds veteran Reserve information) was outside of the block for `militaryServiceEpisodeData` (it lives in a parallel block called `keyData`). So I moved the tag name up one level (called `militaryServiceEpisode`) which encapsulates both previously mentioned blocks. Im not 100% sure if this will have any side-effects but the tests all pass. Just wanted to point it out.

Reply:
https://github.com/department-of-veterans-affairs/vets-api-mockdata/blob/master/emis/military_service_episodes/1007697216.yml